wiki/FitNesseRoot/HsacExamples/SlimTests/BrowserTests/Angular2CalculatorTest.wiki

```diff
@@ -0,0 +1,28 @@
+Sample test on [[Angular][https://angular.io/]] application. 
+
+!note '''Please note:''' The application used in this test does NOT seem to work with !-PhantomJs-!. It does work when executed with Chrome or Firefox.
+
+!note Due to the nature of [[Plunker][https://plnkr.co/]] which hosts the application in an iframe, the page must first be opened with !-BrowserTest-!, switch to the embedded iframe and wait for the application to load.
```

Comment:
Waiting for the async events is indeed why NgBrowserTest was introduced initially. Since then BrowserTest gotten better at waiting for things to be available (if you want to click/check something it will wait for it if it's not there initially), but indeed some things still need it (so far only none of the Angular2 sample sites I found). 

I prefer examples 'from the wild' but to get the sample running we could indeed place it on GitHub pages (or include it in the 'files section' of the wiki and serve via the mock server in the fixtures (i.e. 'mock xml server setup') so we can even use it in the acceptance tests). Having a good check in the acceptance test set would of course be my preferred outcome :-)